### ISSUE TITLE
Remove home page

### ISSUE THREAD

**Issue by pzuraq** [ember-learn/ember-cli-addon-docs#104]
@samselikoff @dfreeman and I discussed this on Slack some time ago but just wanted to capture the convo, the introduction page is fairly redundant now that we have solid Markdown conversion and everything, and it makes more sense to just put people directly into the current `/docs` subroute. This will also make switching between versions easier, and creating the meta-site and navigating between addons on it much easier.

<img width="1003" alt="screen shot 2018-03-15 at 10 10 47 am" src="https://user-images.githubusercontent.com/685518/37479130-23e05060-2839-11e8-9639-8e046c52942d.png">

**Comment by Turbo87** [comment]
I very much support this. This was quite confusing to me when I tried it out yesterday.

**Comment by samselikoff** [comment]
I agree the problem needs to be solved, I'd like to think about the design side for a bit before we move fwd
